### ISSUE TITLE
[question] New Invidious error messages when using mpv to stream links.

### ISSUE THREAD

**Issue by AgentCasey** [iv-org/invidious#4719]
I use mpv to stream many of the video links. I've been noticing errors and failures to play lately and I decided to not ignore them anymore and to see if anyone is aware of this observation.  The version of  my invidious instance is: Current version: 2024.05.27-1ae14cc2 @ master and I admit that I haven't tried to use the same streamed links on any other instances, as of yet, which would probably be a good idea, to localize the concern. 

However, I'll still reach out to see if there is any meaning to this message which I've been receiving lately. I've redacted the IP address for my server and I have substituted the word, "redacted," in its place. Thanks, any insights will be invaluable. 
Here is the text of the error:

Failed to open https://rr4---sn-q4flrn7y.googlevideo.com/videoplayback?expire=1717157263&ei=L2lZZosK6YaS5w-gy4bgAg&ip=redacted&id=o-AGr4bmd7mwFBt_y8RvbseKQyGAMD7vtkbC2e1MxjqYJA&itag=251&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=xT&mm=31%2C29&mn=sn-q4flrn7y%2Csn-q4fl6n6z&ms=au%2Crdu&mv=m&mvi=4&pl=22&initcwndbps=103750&bui=AWRWj2QLdz6vxmogz43SmFU1orvFEy__kASlZO8t13Opke_F0PQ48fPvqd2YOSAxUC1fxb1aQOX_s2Y2&spc=UWF9f4BKm5i_K2TeFkvjpURuZkCAKi8DdCp9vuwJVR8VOdIUZ5XAFeo00P7j&vprv=1&svpuc=1&mime=audio%2Fwebm&ns=vk9hGEtJO7HMF58LK4kjCvQQ&rqh=1&gir=yes&clen=53776652&dur=3296.621&lmt=1717103749038910&mt=1717135271&fvip=4&keepalive=yes&c=WEB&sefc=1&txp=6308224&n=41KYV0_tm6aqKeZ&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cxpc%2Cbui%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cns%2Crqh%2Cgir%2Cclen%2Cdur%2Clmt&sig=AJfQdSswRQIgPfcqGFQdZnU9pI-f4FZRBC5H6FP3FnfAhzSlu9gqyx8CIQC_t9Yzn21Jnuds0yAz9uVELPRDAZiWrpalaH05HhyzBA%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AHWaYeowRAIgOhS7hi6ENk7bP1jhGiTGsHOxmiVF5rJWST1rRHmYA_gCIBswh6_j9XCpENwvrD_W4cigtHt8Fc09u_yFUxHqotuQ.

**Comment by unixfox** [comment]
Make sure you are not trying to load copyrighted video like music.

**Comment by AgentCasey** [comment]
@{https://github.com/unixfox} No I'm not trying to access any copyrighted videos, It's a recent phenomenon and I usually stream all my videos from my own instance. However, I've discovered that the same message is appearing when I use other instances to prove that it's not just my own instance. I may try using older binaries of invidious to see if it's related to the latest upgrade or perhaps, maybe YouTube and Google are up to their usual tricks.

**Comment by AgentCasey** [comment]
I've used previous binaries, older ones which I've kept after the building process and the error message keeps coming back occasionally. After a few tries the video will eventually play but one has to be attended. That is, when playlists are made, the error will cause interruptions. 

The error does not seem to affect playback in the browsers however, which is why it will be overlooked by many people. However, if it is Google finding new ways of subversion, I could use some eyes to interpret the error.  Thanks.
